erts/doc/src/erl_cmd.xml

```diff
@@ -1870,14 +1875,15 @@ $ <input>erl \
       <tag>The <c>.erlang</c> startup file</tag>
       <item>
         <p>When Erlang/OTP is started, the system searches for a file named
-        <c>.erlang</c> in the user&apos;s home directory.</p>
+          <c>.erlang</c> in the user&apos;s home directory or
+          <seemfa marker="stdlib:filename#basedir/3"><c>$XDG_CONFIG_HOME/erlang</c></seemfa>.</p>
```

Comment:
What if `.erlang` files exist both places? Should the search order be documented?